convert2rhel/checks.py

```diff
@@ -422,10 +422,21 @@ def _bad_kernel_version(kernel_release):
 
 def _bad_kernel_package_signature(kernel_release):
     """Return True if the booted kernel is not signed by the original OS vendor, i.e. it's a custom kernel."""
-    kernel_pkg = run_subprocess(
+    kernel_pkg, return_code = run_subprocess(
         ["rpm", "-qf", "--qf", "%{NAME}", "/boot/vmlinuz-%s" % kernel_release], print_output=False
-    )[0]
+    )
     logger.debug("Booted kernel package name: {0}".format(kernel_pkg))
+
+    vmlinuz_path = "/bootvmlinuz-%s" % kernel_release
+
+    if return_code == 0:
+        logger.warning(
+            "The booted kernel %s is not owned by any installed package."
+            " It needs to be owned by a package signed by %s." % (vmlinuz_path, os_vendor)
```

Comment:
`os_vendor` is undefined. Can see how to fetch it from here 
https://github.com/oamg/convert2rhel/blob/120ad4f6356f50e861acc15f826132c4c9d0e7e3/convert2rhel/checks.py?rgh-link-date=2022-02-16T15%3A42%3A26Z#L432-L434

Reply:
I'll implement this definition to my code to get this fixed, thank you for the feedback!